src/gfx/context.cpp

```diff
@@ -27,7 +29,10 @@ static inline std::shared_ptr<spdlog::logger> &getLogger() {
 #ifdef WEBGPU_NATIVE
 static WGPUBackendType getDefaultWgpuBackendType() {
 #if GFX_WINDOWS
-  return WGPUBackendType_D3D12;
```

Comment:
Maybe revert this? Anyway let's keep a closer look I guess. Maybe need to test both in CI too

Reply:
Any idea why runs slow? Cos of some mistakes they did like you said last time?

Reply:
Yeah, one of those open issues about fixing D3D12. Copying data to buffers is currently very slow